SCClassLibrary/Common/Math/Rational.sc

```diff
@@ -7,23 +7,25 @@ Rational : Number {
 
 	reduce {
 		var d;
```

Comment:
this method sometimes returns a new instance, sometimes modifies the old one. Which is better? Generally, returning a new object is preferable, but it depends.